index.html

```diff
@@ -42,205 +59,428 @@ <h1>I'm Joseph Glad to meet you!</h1>
+    
+    
+            </ul>
+    
+            <button class="about-button-desktop"><a href="">Get my resume</a></button>
         </div>
-        
-        <ul class="link">
-            <li><a href="">Html</a></li>
-            <li><a href="">css</a></li>
-            <li><a href="">javascript</a></li>
-         </ul>
+        <div class="aside-about-desktop">
 
-         <button class="button"><a href="">See Project</a></button>
-        
-    </section>
-    <section class="grid-item">
-        <img class="snap" src="Images/Snapshoot Portfolio (3).png" alt="">
-        <h2 class="multi">Multi-Post
-            Stories</h2>
-        <div class="container">
-            <div class="canopy">
-                <p>CANOPY</p>
-            </div>
-            
-            <div class="details">
-                <img src="Images/Counter.png" alt="">
-                <p>Backend Dev</p>
+            <div class="lang-desktop">
+                <h3>Languages</h3>
+                <img src="Images/arrow-down-sign-to-navigate.png" alt="">
             </div>
-            <div class="details">
-                <img src="Images/Counter.png" alt="">
-                <p>2015</p>
-            </div>
-        </div>
-        <div class="content">
-            <p>
-                A daily selection of privately personalized reads; 
-                no accounts or sign-ups required.
-            </p>
+    
+            <ul class=" basic-desktop">
+                <li class="stack-desktop"><img src="Images/Ellipse 1.png" alt="">
+                    <p>Javascript</p>
+                </li>
+                <li class="stack-desktop"><img src="Images/Ellipse 1 (1).png" alt="">
+                    <p>HTML</p>
+                </li>
+                <li class="stack-desktop"><img src="Images/Ellipse 1 (2).png" alt="">
+                    <p>CSS</p>
+                </li>
+            </ul>
+    
+            <ul class="lang-skills-desktop">
+                <li class="framework-desktop">
+                   <a href="">Frameworks</a>  <img class="image-desktop" src="Images/right-arrow.png" alt="">
+                </li>
+                <li><img class="shape-desktop" src="Images/Separator Bottom.png" alt=""></li>
+                <li class="framework-desktop">
+                   <a href="">Skills</a>  <img class="image-desktop" src="Images/right-arrow.png" alt="">
+                </li>
+                <li><img class="shape-1-desktop" src="Images/Separator Bottom.png" alt=""></li>
+            </ul>
+    
         </div>
-        
-        <ul class="link">
-            <li><a href="">Html</a></li>
-            <li><a href="">css</a></li>
-            <li><a href="">javascript</a></li>
-         </ul>
-
-         <button class="button"><a href="">See Project</a></button>
-        
+        </section>   
+    </div>
+  
+    <!-- This is the contact page -->
+    <section id="contact">
+        <h2>Contact Me</h2>
```

Comment:
- [ ] I can see you made  a beautiful design here but you can still improve it for it to match the Figma design , I just noticed that your background is not well set , kindly fix it for your website to look more professional :smile: 
Your Design 
![Screenshot from 2023-03-02 15-57-51](https://user-images.githubusercontent.com/88894525/222450291-5d1fbdd2-4d80-4c26-89c5-154b5df84fad.png)

Figma Design 
![Screenshot from 2023-03-02 16-00-30](https://user-images.githubusercontent.com/88894525/222450541-720d4213-8024-4e21-962f-a428df978503.png)

_note: You misspell Contact me , I think the  "Me" should be "me" m in lower case_